### PR TITLE
updated sentance case

### PR DESCRIPTION
**Pull request by Juanro-Minne**
image
image
image

- labels now have correct camel case
- Updated cellphone number field to have correct casing and text
- updated error message

### CODE REVIEW

**Review by MathildaShongwe15 — DISMISSED** [review]

File: src/app/components/hris/employees/new-employee/new-employee.component.html [hunk 1]
```diff
@@ -100,11 +100,11 @@ <h1 class=" mb-0 mb-md-1" id="heading">Employee Details</h1>
                     (input)="onCellphoneInputCheck(inputField.value ?? '')"
                   ></ngx-mat-intl-tel-input>
                   <mat-error *ngIf="newEmployeeForm.controls['cellphoneNo'].hasError('required') && newEmployeeForm.controls['cellphoneNo'].value=='' ">
-                    Phone Number Is Required
+                    Cellphone Number Is Required
                   </mat-error>
                   <mat-error
                     *ngIf="newEmployeeForm.controls['cellphoneNo'].value != '' && newEmployeeForm.controls['cellphoneNo'].invalid">
-                    This Is Not a Valid Phone Number
+                    This Is Not a Valid Cellphone Number
```
Comment (madwizzard) [inline src/app/components/hris/employees/new-employee/new-employee.component.html:107]: This is not a valid Cellphone number

File: src/app/components/hris/employees/employee-profile/accordions/accordion-career-summary/accordion-certificates/accordion-certificates.component.html [hunk 2]
```diff
@@ -54,14 +54,14 @@ <h3 class="mb-1" id="certificateNameHeading">{{sharedAccordionFunctionality.empl
...
-                                            <mat-label>Name of organization</mat-label>
+                                            <mat-label>Name Of Organisation</mat-label>
                                             <input type="text" matInput placeholder="Name of organization"
                                                 [(ngModel)]="copyOfCertificates[i].issueOrganization"
                                                 [value]="copyOfCertificates[i].issueOrganization">
                                         </mat-form-field>
                                         <ng-template #NoEditOrganization>
                                             <mat-form-field appearance="outline" class="w-100">
-                                                <mat-label>Name of organization</mat-label>
+                                                <mat-label>Name Of organisation</mat-label>
```
Comment (MathildaShongwe15) [inline src/app/components/hris/employees/employee-profile/accordions/accordion-career-summary/accordion-certificates/accordion-certificates.component.html:64]: organisation has to be title case

Comment (Juanro-Minne) [reply src/app/components/hris/employees/employee-profile/accordions/accordion-career-summary/accordion-certificates/accordion-certificates.component.html]: done

File: src/app/components/hris/employees/new-employee/new-employee.component.html (hunk 1 above)
Comment (Juanro-Minne) [reply src/app/components/hris/employees/new-employee/new-employee.component.html]: updated

File: src/app/components/hris/employees/new-employee/new-employee.component.html [hunk 3]
```diff
@@ -100,11 +100,11 @@ <h1 class=" mb-0 mb-md-1" id="heading">Employee Details</h1>
                     (input)="onCellphoneInputCheck(inputField.value ?? '')"
                   ></ngx-mat-intl-tel-input>
                   <mat-error *ngIf="newEmployeeForm.controls['cellphoneNo'].hasError('required') && newEmployeeForm.controls['cellphoneNo'].value=='' ">
-                    Phone Number Is Required
+                    Cellphone Number Is Required
                   </mat-error>
                   <mat-error
                     *ngIf="newEmployeeForm.controls['cellphoneNo'].value != '' && newEmployeeForm.controls['cellphoneNo'].invalid">
-                    This Is Not a Valid Phone Number
+                    This is not a valid Cellphone number.
```
Comment (MathildaShongwe15) [inline src/app/components/hris/employees/new-employee/new-employee.component.html:107]: image

Comment (Juanro-Minne) [reply src/app/components/hris/employees/new-employee/new-employee.component.html:107]: checking with wouter

Comment (Juanro-Minne) [reply src/app/components/hris/employees/new-employee/new-employee.component.html]: This is not a valid cellphone number.  this is what wouter wants

**Review by MathildaShongwe15 — APPROVED** [review]

**Review by ReinoRabbit — APPROVED** [review]

**Review by MathildaShongwe15 — APPROVED** [review]

**Review by EduanO — APPROVED** [review]

**Review by madwizzard — APPROVED** [review]

**Pull request merged**